changeset/commit.go

```diff
@@ -31,6 +31,12 @@ var (
 	once      sync.Once
 
 	readBuildInfo = debug.ReadBuildInfo
+
+	// Revision contains the revision of the binary.
+	// It is exported so that it can be overriden using ldflags.
+	// By default, it will use 'vcs.revision' property from the embedded build information
+	// Note: Use changeset.Get() to get it instead of through this variable.
```

Comment:
Can ldflags only set an exported var? Can we just unexport this?

Reply:
hum that was my initial though (because it's been a *long* time since I did that…). Let me test it out.

Reply:
ah seems like it works with unexported, let me update this.